changelogs/unreleased/66827.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "SaleOrderLine : Project not emptied on copy"
```

Comment:
Please make the title more explicit (can be interpreted as, "made a fix so project are not emptied on copy")
For example: "SaleOrderLine: Fixed a bug where project was not emptied on copy"